testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -953,15 +956,62 @@ public void getRowsWithInQuery() throws IOException {
         RestUtils.get(
             authToken,
             String.format(
-                "%s:8082/v2/keyspaces/%s/%s?where=%s&raw=true",
-                host, keyspaceName, tableName, whereClause),
+                "%s/v2/keyspaces/%s/%s?where=%s&raw=true",
+                restUrlBase, keyspaceName, tableName, whereClause),
             HttpStatus.SC_OK);
 
     List<Map<String, Object>> data =
         objectMapper.readValue(body, new TypeReference<List<Map<String, Object>>>() {});
     assertThat(data.size()).isEqualTo(1);
     assertThat(data.get(0).get("id")).isEqualTo("1");
     assertThat(data.get(0).get("firstName")).isEqualTo("Sarah");
+
+    // Let's also test with three values (of which 2 match)
+    whereClause =
+        "{\"id\":{\"$eq\":\"1\"},\"firstname\":{\"$in\":[\"Sarah\", \"Bob\", \"John\" ]}}";
+    body =
+        RestUtils.get(
+            authToken,
+            String.format(
+                "%s/v2/keyspaces/%s/%s?where=%s&raw=true",
+                restUrlBase, keyspaceName, tableName, whereClause),
+            HttpStatus.SC_OK);
+
+    JsonNode root = objectMapper.readTree(body);
+    Set<String> namesReceived =
+        new LinkedHashSet<>(
+            Arrays.asList(
+                root.path(0).path("firstname").asText(), root.path(1).path("firstname").asText()));
+    Set<String> namesExpected = new LinkedHashSet<>(Arrays.asList("Sarah", "John"));
+    assertThat(namesReceived).isEqualTo(namesExpected);
+  }
+
+  @Test
+  public void getRowsWithExistsQuery() throws IOException {
+    createKeyspace(keyspaceName);
+    createTestTable(
+        tableName,
+        Arrays.asList("id text", "firstName text", "lastName text", "extra text"),
+        Collections.singletonList("id"),
+        Collections.singletonList("firstName"));
+
+    insertTestTableRows(
+        Arrays.asList(
+            Arrays.asList("id 1", "firstName Bob"),
+            Arrays.asList("id 1", "firstName Dave"),
+            Arrays.asList("id 1", "firstName Jack", "lastName Daniels")));
+
+    // First let's try to find something that does not exist
+    String whereClause = "{\"id\":{\"$eq\":\"1\"},\"extra\":{\"$exists\":true}}";
+    String body =
+        RestUtils.get(
+            authToken,
+            String.format(
+                "%s/v2/keyspaces/%s/%s?where=%s&raw=true",
+                restUrlBase, keyspaceName, tableName, whereClause),
+            HttpStatus.SC_OK);
```

Comment:
Here we get odd failure:

```
{"description":"Bad request: Invalid value provided for 'extra': Java value true of type 'java.lang.Boolean' is not a valid value for CQL type varchar","code":400}
```

which looks a lot like a bug. But I want to sanity check it first in case I just wrote bad query :)

Reply:
Apparently this just is the way things work for now; see #1519 for follow-up discussion.